apps/api/src/domain/domainModelValidators/__tests__/domainModelValidators/buildDomainModelValidatorTestCases/bibliographic-reference.domainModelValidatorTestCase.ts

```diff
@@ -0,0 +1,49 @@
+import { IBibliographicReference } from '../../../../models/bibliographic-reference/interfaces/IBibliographicReference';
+import { BibliographicReferenceType } from '../../../../models/bibliographic-reference/types/BibliographicReferenceType';
+import { resourceTypes } from '../../../../types/resourceTypes';
+import bibliographicReferenceValidator from '../../../bibliographicReferenceValidator';
+import InvalidEntityDTOError from '../../../errors/InvalidEntityDTOError';
+import { DomainModelValidatorTestCase } from '../../types/DomainModelValidatorTestCase';
+import { buildBookBibliographicReferenceTestCases } from './bibliographicReferenceInvalidTestCases/bookBibliographicReference.domainModelValidatorTestCases';
+import getValidBibliographicReferenceInstanceForTest from './utils/getValidBibliographicReferenceInstanceForTest';
+
+const validCase = {
+    description: 'When given a valid book bibliographic reference DTO',
+    dto: getValidBibliographicReferenceInstanceForTest(BibliographicReferenceType.book),
+};
+
+const modelSpecificTestCases = [...buildBookBibliographicReferenceTestCases()];
+
+export const buildBibliographicReferenceTestCase =
+    (): DomainModelValidatorTestCase<IBibliographicReference> => ({
+        resourceType: resourceTypes.bibliographicReference,
+        validator: bibliographicReferenceValidator,
+        validCases: [validCase],
```

Comment:
When adding the next sub-type of `Bibliographic Reference`, be sure to add an additional valid case as well.